.github/workflows/validate_alerts_format.yml

```diff
@@ -0,0 +1,53 @@
+name: validate format of alerts changes
```

Comment:
nit: Validate alerts metadata

Reply:
The script validates metadata and jsons - should we still update the name?

Reply:
Ah, maybe "Validate alert templates and metadata" then. I would just prefer it be more specific about what is being validated.

Reply:
Gotcha. Updated.